yoti_python_sdk/dynamic_sharing_service/share_url.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import json
+
+INVALID_DATA = "Json is incorrect, contains invalid data"
+APPLICATION_NOT_FOUND = "Application was not found"
+UNKNOWN_ERROR = "Unknown HTTP error occured"
+
+
+def create_share_url(yoti_client, dynamic_scenario):
+    http_method = "POST"
+    payload = json.dumps(dynamic_scenario, sort_keys=True).encode()
+    endpoint = yoti_client.endpoints.get_dynamic_share_request_url()
+    response = yoti_client.make_request(http_method, endpoint, payload)
+
+    status_code = response.status_code
+    if status_code >= 200 and status_code < 300:
```

Comment:
Is this a bit clearer and more concise? 
```suggestion
    if 200 <= status_code < 300:
```

Reply:
Oo. I didn't know you could do that.

Reply:
[PyCharm](https://www.jetbrains.com/pycharm/download) (community edition is free) suggests things like that automatically, if you ever want to use anything but Vim 😃